pages/projects.tsx

```diff
@@ -9,10 +9,9 @@ import {
   ChildButton,
   Directions,
 } from 'react-floating-button-menu';
-import { projects } from '../sample-data/data';
 import { Icon } from '@iconify/react';
 import 'twin.macro';
-
+import {projects} from "../sample-data/data";
```

Comment:
i think it's still getting projects from the local json data. could you make it use the mongodb data?

Reply:
Hi @rohitdasu , I want to remove local json data, but I am not able to retrieve mongo data on above file(repo.project.ts) in function(findProjects), but not able to export that data to this file(projects.tsx), Could you please guide how to do that.

Reply:
sure got it. I will help you. Firstly please pull the latest code from main branch and fix the conflicts.

Reply:
okay I will.